packages/app/src/actions/emails.js

```diff
@@ -0,0 +1,48 @@
+import ErrorActions from './errors'
+import * as ActionTypes from '../actions/types'
+
+const SERVER_URL = process.env.REACT_APP_SERVER_URL
+
+const EmailActions = {
+
+  sendEmails(params) {
+    return async function (dispatch) {
+      try {
+        const decoder = new TextDecoder('utf-8')
+        const response = await fetch(`${SERVER_URL}/emails`, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          credentials: 'include',
+          body: JSON.stringify(params),
+        })
```

Comment:
What do you think if we use [this](https://github.com/aragonone/court-backend/blob/development/packages/app/src/web3/Server.js) guy?

Reply:
I tried, but axios don't do streaming in the frontend